app/views/vacancies/_school.js.haml

```diff
@@ -3,7 +3,7 @@
     var myLatLng = {lat: #{lat}, lng: #{lng}};
 
     var map = new google.maps.Map(document.getElementById('map_zoom'), {
-      zoom: 17,
+      zoom: 16,
```

Comment:
This level of zoom gives a better view of where the school is in context: not just the few streets around it.